src/content/en/fundamentals/architecture/howto-components/howto-checkbox.md

```diff
@@ -0,0 +1,296 @@
+project_path: /web/_project.yaml
+book_path: /web/fundamentals/_book.yaml
+
+{# wf_updated_on: 2017-04-26#}
+{# wf_published_on: 2017-04-06 #}
+
+# HowTo: Components – howto-checkbox {: .page-title }
+
+{% include "web/_shared/contributors/ewagasperowicz.html" %}
+{% include "web/_shared/contributors/noms.html" %}
+{% include "web/_shared/contributors/robdodson.html" %}
+{% include "web/_shared/contributors/surma.html" %}
+
+<link rel="stylesheet" href="prism-solarizedlight.css">
+<link rel="stylesheet" href="main.css">
+
+<p>A <code>HowToCheckbox</code> represents a boolean option in a form. The most common type
```

Comment:
Try to avoid mixing markdown and HTML. DevSite's markdown parser is old and gets easily confused. Preference for Markdown.